src/MainWindow.h

```diff
@@ -158,9 +158,10 @@ private slots:
 
 private slots:
 	void selectFindType(int);
-	void find();
-	void findString(QString);
-	void findAndReplace();
+    void hideFind();
```

Comment:
Feel free to update any other inconsistencies - I tend to only point out the ones I can see in diffs.